Google Cloud/google-drive-reports/ingest/parser.yml

```diff
@@ -0,0 +1,31 @@
+name: google-drive-reports
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+
+  - name: set_ecs_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          event.kind: "event"
+          event.category: ["file"]
+          event.type: ["access"]
```

Comment:
According to the `\"events\":[{\"type\":\"access\",\"name\":\"edit\"` in the examples provided, I think `event.type` should be in this case `["access", "edit"]`